assets/src/components/login/Login.tsx

```diff
@@ -41,16 +41,18 @@ const setInputFocus = (ref: RefObject<any>) => {
   })
 }
 
-function LoginError({ error }) {
+function LoginError({ error }: { error: ApolloError | undefined }) {
   useEffect(() => {
-    const to = setTimeout(() => {
-      wipeToken()
-      wipeRefreshToken()
-      window.location = '/login' as any as Location
-    }, 2000)
+    if (error?.networkError?.message !== 'Failed to fetch') {
```

Comment:
i think it's ok to ignore if it's a network error entirely although do 500's get parsed as network errors?

There's actually a couple discrete errors we send when it doesn't auth:

* if there is no token, you'll get an `unauthenticated` error message
* if it's an expired/invalid jwt, you'll probably get a `401` with an `unauthorized` response.